lib/api/core/entrypoints/embedded/protocols/websocket.js

```diff
@@ -52,12 +53,27 @@ class WebsocketProtocol extends Protocol {
    */
   init (entryPoint) {
     super.init(entryPoint);
```

Comment:
Nitpicking: no need to init the superclass if the protocol is disabled. So we could move this line after the next conditional statement ?

Reply:
Good point. I made the change for other protocols as well, as this can be confusing to have a debug message saying that we're initializing an actually disabled protocol.